qcodes/instrument_drivers/Minicircuits/USB_SPDT.py

```diff
@@ -66,8 +66,12 @@ def __init__(self, name: str, driver_path: Optional[str] = None,
                 the dll to open properties and check the 'unblock' checkmark
                 in the bottom. Check that your python installation is 64bit."""
             )
-        import mcl_RF_Switch_Controller64
-        self.switch = mcl_RF_Switch_Controller64.USB_RF_SwitchBox()
+        try:
+            import mcl_RF_Switch_Controller64 as mw_driver
+        except:
```

Comment:
Could we add the specific type of exception that we want to catch here for clarity?

Reply:
I think the try-except statement is Pythonic here so would be fine to go with that, it's just good practice to also catch the actual exception. It could be that this import throws a different exception, even on a system that has the older .NET installed and that will then fail in an unclear way because we didn't catch the exception.